### PR TITLE
Release v1.0.4

### PR DESCRIPTION
**Pull request by PabloReszczynski**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@topsort/analytics.js",
-  "version": "1.0.3",
+  "version": "1.1.4",
```
Comment (sk-) [inline package.json:3]:
```suggestion
  "version": "1.0.4",
```

**Review by sk- — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (PabloReszczynski) [reply package.json:3]: should the minor version reset to 0 in that case?

Comment (PabloReszczynski) [reply package.json:3]: Woops, my bad

Comment (sk-) [reply package.json]: According to https://semver.org/ 
> MINOR version when you add functionality in a backwards compatible manner, and
> PATCH version when you make backwards compatible bug fixes.

So in this case I think it's appropriate to make a patch version.

**Pull request merged**